### DISCUSSION TITLE
Import saves from original gothic 2

### DISCUSSION THREAD

**Discussion by Laob** [Try/OpenGothic#667] [Q&A] [answered]
I want to switch from my PC to my Mac M1 to play Gothic 2 and I want to ask, whether it is possible to import the saves from the original gothic 2 to opengothic. I tried to copy the saves to the OpenGothic folder in Application Support, but as the original gothic has multiple files for saving one slot and opengothic only one, I'm not sure, if it is possible to import them. Thank you in advanced!

**Comment by Abendlied** [comment, marked as answer]
Hi @Laob,
currently the savegames are incompatible, OpenGothic uses a specific format. You cannot import original Gothic 2 savegame files to OpenGothic at this time.

I believe I have read somewhere that the most practical approach would be to create a standalone savegame converter. Though I am unaware if there are any actual plans.

**Reply by Laob** [reply]
Hi Abendlied! Thank you for your response 👍

**Comment by Try** [comment]
[ZenKit](https://github.com/GothicKit/ZenKit) has some support for vanilla save-game, but I don't know if it's finished or not.
Issue for save-game support in opengothic: https://github.com/Try/OpenGothic/issues/410

In principle save-game are not very different to world-files, that opengothic is already de-facto supports, so should be doable
